<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test1Dialog</class>
 <widget class="QDialog" name="Test1Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Діяльнісно-операційний тест</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1053</width>
        <height>1137</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>4.  Чи вмієте Ви в цікавій та доступній формі надавати інформацію клієнтові?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="4" column="0">
           <widget class="QRadioButton" name="rb43">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb42">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb41">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="rb44">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>7. Чи володієте Ви автоматизованими системами пошуку, онлайн-бронювання та купівлі залізничних, автобусних, авіаквитків?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb71">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb72">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb73">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb74">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>5.  Чи вмієте Ви оформляти договори і контракти з реалізації турів?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb51">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb52">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb53">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb54">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>1. Чи вмієте Ви розробляти й оновлювати інформаційний рекламний матеріал?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb11">
            <property name="text">
             <string>З використанням засобів розробки веб-сайтів</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb12">
            <property name="text">
             <string>З використанням текстового редактора</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb13">
            <property name="text">
             <string>З використанням графічних редакторів</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb14">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>6. Чи вмієте Ви збирати й оброблювати інформацію із застосуванням спеціалізованого (комп’ютерного) програмного забезпечення?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb61">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb62">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb63">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb64">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>3. Чи вмієте Ви організовувати та планувати подорожі (екскурсії)?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb31">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="rb34">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb33">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb32">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>rb31</zorder>
         <zorder>rb32</zorder>
         <zorder>rb33</zorder>
         <zorder>rb34</zorder>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>9. Чи володієте Ви автоматизованими системами пошуку, онлайн-бронювання та купівлі туру, оформлення, збереження та відправки замовлення туроператору, відстеження стану замовлення?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb91">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb92">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb93">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb94">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>2. Чи володієте Ви уміннями отримувати інформацію про місце розташування, клімат, історію країн світу?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="5" column="0">
           <widget class="QRadioButton" name="rb24">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb22">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb23">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb21">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>8. Чи володієте Ви автоматизованими системами пошуку, онлайн-бронювання та купівлі номеру в готелі?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rb81">
            <property name="text">
             <string>Так</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rb82">
            <property name="text">
             <string>Ні</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rb83">
            <property name="text">
             <string>Частково</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rb84">
            <property name="text">
             <string>Вагаюсь відповісти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Test1Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Test1Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
